app-admin/lnav/metadata.xml

```diff
@@ -1,14 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="person" proxied="yes">
-    <email>randy@electronsweatshop.com</email>
-    <name>Randy Barlow</name>
-  </maintainer>
-  <maintainer type="project" proxied="proxy">
-    <email>proxy-maint@gentoo.org</email>
-    <name>Proxy Maintainers</name>
-  </maintainer>
```

Comment:
I assumed I shouldn't leave the proxy maintainer group here, but I'm happy to bring it back if we want to keep that.